app/src/androidTest/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/splash/SplashScreenTest.kt

```diff
@@ -26,26 +23,34 @@ const val EMAIL = "example@gmail.com"
 const val VALID_PASSWORD = "valid"
 const val INVALID_PASSWORD = "invalid"
 const val ERROR_MESSAGE = "error"
+private val SPLASH_TIME = 0L
```

Comment:
We can make this a constant 😄 
```suggestion
private const val SPLASH_TIME = 0L
```